### PR TITLE
added new route in news to provide snippets to frontend in json

### PR DESCRIPTION
**Pull request by AASHRITH1903**

### CODE REVIEW

File: routes/news.js [hunk 1]
```diff
@@ -72,4 +72,13 @@ router.route('/view_all').get(adminAuth, (req, res) => {
     })
 })
 
+router.route('/front').get(adminAuth, (req,res) => {
```
Comment (shobhi1310) [inline routes/news.js:75]: why is their an authentication middleware?
there should be no authentication middleware for this.

File: routes/news.js [hunk 2]
```diff
@@ -72,4 +72,13 @@ router.route('/view_all').get(adminAuth, (req, res) => {
...
+    newsModel.find()
```
Comment (shobhi1310) [inline routes/news.js:76]: use findOne

File: routes/news.js [hunk 3]
```diff
@@ -72,4 +72,13 @@ router.route('/view_all').get(adminAuth, (req, res) => {
...
+    .then(newss => {
```
Comment (shobhi1310) [inline routes/news.js:77]: news itself is plural write 'news'

File: routes/news.js (hunk 1 above)
Comment (AASHRITH1903) [reply routes/news.js:75]: ok

**Pull request merged**